### ISSUE TITLE
custom fields

### ISSUE THREAD

**Issue by jneiva08** [openscopeproject/InteractiveHtmlBom#294]
Hi,

When try to create ibom from kicad v6.x.x it doesn't appear custom fields.

symbol field table from kicad:
![symbol field table](https://user-images.githubusercontent.com/8236353/163141343-a4acf3e1-c2c7-4c76-bd12-1f7bf84eaeba.png)

When try generate the ibom in the fields tab it doesn't show the custom field.
![ibom_fileds](https://user-images.githubusercontent.com/8236353/163141305-1e88e0e1-69e6-401b-839c-1d6ba40c1311.png)

What can do to fix?

**Comment by qu1ck** [comment]
How did you install the plugin? How did you launch it?
There are few odd things in your screenshots: the window shows version 6.0.0 of the plugin which doesn't exist. It also didn't load the pcb file in extra data field, it should have.

See if manually selecting extra data file (choose the same pcb file you are generating bom for) helps.

**Comment by jneiva08** [comment]
I have Linux Mint 20, the kicad version is 6.0.4-6f826c9f35~116~ubuntu20.04.1, release build
I have installed the ibom by "Plugin And Content Manager"

In my work computer have the version displayed.
I have installed in my home computer and have the right version.

![image](https://user-images.githubusercontent.com/8236353/163687418-d84207aa-c1e8-4f36-90bf-722ec95df017.png)

The image was from a project made in kicad v5, in the first time didn't show lcsc and manf#.
In the image appear because have "Update PCB with changes made to schematic" and appears.
